core/src/main/java/org/lflang/generator/c/CGenerator.java

```diff
@@ -710,7 +712,8 @@ private void inspectReactorEResource(ReactorDecl reactor) {
       if (lfResource != null) {
         var config = lfResource.getTargetConfig();
         // FIXME: this should not happen here, but once, after collecting all the files.
-        copyUserFiles(config, lfResource.getFileConfig());
+        // copyUserFiles(config, lfResource.getFileConfig());
```

Comment:
```suggestion
```

Reply:
This seems to have already been resolved.